### PR TITLE
Change `Style/RedundantFreeze` message

### PR DESCRIPTION
**Pull request by lucasuyezu**
This commit changes the message for `Style/RedundantFreeze`. The new message is less aggressive and is more uniform with other messages in the code base, like [`Lint/HandleExceptions`](https://github.com/bbatsov/rubocop/blob/master/lib/rubocop/cop/lint/handle_exceptions.rb#L8).

-----------------

Before submitting the PR make sure the following are checked:

* [x] Wrote [good commit messages][1].
* [x] Commit message starts with `[Fix #issue-number]` (if the related issue exists).
* [x] Used the same coding conventions as the rest of the project.
* [x] Feature branch is up-to-date with `master` (if not - rebase it).
* [x] Squashed related commits together.
* [x] Added tests.
* [x] Added an entry to the [Changelog](../blob/master/CHANGELOG.md) if the new code introduces user-observable changes. See [changelog entry format](../blob/master/CONTRIBUTING.md#changelog-entry-format).
* [x] All tests are passing.
* [x] The new code doesn't generate RuboCop offenses.
* [x] The PR relates to *only* one subject with a clear title
  and description in grammatically correct, complete sentences.
* [x] Updated cop documentation with `rake generate_cops_documentation` (required only when you've added a new cop or changed the configuration/documentation of an existing cop).

[1]: http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html

### CODE REVIEW

File: lib/rubocop/cop/style/redundant_freeze.rb [hunk 1]
```diff
@@ -14,7 +14,7 @@ module Style
       class RedundantFreeze < Cop
         include FrozenStringLiteral
 
-        MSG = 'Freezing immutable objects is pointless.'.freeze
+        MSG = "Do not freeze immutable objects.".freeze
```
Comment (kmcphillips) [inline lib/rubocop/cop/style/redundant_freeze.rb]: How about:

> Freezing immutable objects has no effect.

That explains the _why_ as well as giving the instruction.

Comment (bbatsov) [inline lib/rubocop/cop/style/redundant_freeze.rb]: Or "Do not freeze immutable objects, as freezing them has no effect.".

**Comment by lucasuyezu** [comment]
@bbatsov Message updated.

**Comment by bbatsov** [comment]
You've broken the build by changing the style of the string quotes. You'll also need to rebase.

**Comment by bbatsov** [comment]
The build is still failing.

**Pull request merged**